### PR TITLE
Add a sample of simple echo.

### PR DESCRIPTION
**Pull request by LGDoor**

### CODE REVIEW

File: samples/SimpleEcho/Startup.cs [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+        {
+            app.UseAzureSignalR(routes =>
+            {
+                routes.MapHub<EchoHub>("/echo");
+            });
+            applicationLifetime.ApplicationStarted.Register(async () =>
+            {
+                try
+                {
+                    var task = new Client().Run();
```
Comment (chenkennt) [inline samples/SimpleEcho/Startup.cs]: Does this always get executed after server is connected to the service?

Comment (LGDoor) [reply samples/SimpleEcho/Startup.cs]: Yes this function is invoked after all services started.

File: samples/SimpleEcho/Client.cs [hunk 2]
```diff
@@ -0,0 +1,30 @@
+﻿using System;
```
Comment (xscript) [inline samples/SimpleEcho/Client.cs]: Licence header

Comment (xscript) [reply samples/SimpleEcho/Client.cs]: Add it for all `cs` files

File: samples/SimpleEcho/SimpleEcho.csproj [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Folder Include="wwwroot\" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.AspNetCore.App" />
+    <PackageReference Include="Microsoft.AspNetCore.SignalR.Client" Version="1.0.0" />
+    <PackageReference Include="Microsoft.Azure.SignalR" Version="1.0.0-preview1-10011" />
```
Comment (xscript) [inline samples/SimpleEcho/SimpleEcho.csproj]: Let's use `1.0.0-*` to always reference the latest package.

File: samples/SimpleEcho/Startup.cs [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+            services.AddSignalR().AddAzureSignalR();
+        }
+
+        public void Configure(IApplicationBuilder app, IHostingEnvironment env, IApplicationLifetime applicationLifetime)
+        {
+            app.UseAzureSignalR(routes =>
+            {
+                routes.MapHub<EchoHub>("/echo");
+            });
+            applicationLifetime.ApplicationStarted.Register(async () =>
```
Comment (xscript) [inline samples/SimpleEcho/Startup.cs]: How about extracting a method instead of using anonymous method for better readability?

**Pull request merged**